src/app/components/LiveLabel/index.styles.ts

```diff
@@ -1,13 +1,65 @@
-import { css, Theme } from '@emotion/react';
+import { css, Theme, keyframes } from '@emotion/react';
+
+const pulseAnimation = keyframes({
+  '0%': {
+    transform: 'scale(1)',
+    opacity: 1,
+  },
+  '50%': {
+    transform: 'scale(1)',
+    opacity: 0,
+  },
+  '100%': {
+    transform: 'scale(1)',
+    opacity: 1,
+  },
+});
 
 const styles = {
-  liveLabel: ({ palette, spacings, fontVariants }: Theme) =>
+  liveLabelText: ({ palette, spacings, fontVariants }: Theme) =>
     css({
       color: palette.LIVE_DARK,
       display: 'inline-block',
       marginInlineEnd: `${spacings.FULL}rem`,
       ...fontVariants.sansBold,
     }),
+  liveLabelCircle: ({ palette, spacings }: Theme) =>
+    css({
+      color: palette.LIVE_DARK,
+      borderRadius: '50%',
+      display: 'inline-block',
+      width: '.9375rem',
+      height: '.9375rem',
+      background:
+        'radial-gradient(circle,transparent 53%,hsla(180, 100%, 20%,.5) 53.5%,#006666 54%)',
+      marginInlineEnd: `${spacings.FULL}rem`,
+      position: 'relative',
+      // transform: 'translateX(-50%) translateY(-50%)',
+      // left: '50%',
+      // right: '50%',
+    }),
+
+  livelabelPulse: () =>
+    css({
+      '&:before': {
+        content: `''`,
+        position: 'absolute',
+        width: '100%',
+        height: '100%',
+        borderRadius: '50%',
+        background:
+          'radial-gradient(circle,#006666 37%,rgba(0,102,102,.5) 37.5%,transparent 38%)',
```

Comment:
As above, let's not hardcode the colour here, and just use the colour from the palette.